dallinger/experiment_server/experiment_server.py

```diff
@@ -1435,4 +1450,5 @@ def insert_mode(page_html, mode):
             page_html[match.end():]
         return new_html
     else:
+        traceback.print_exc()
```

Comment:
I don't remember it, but yeah, I think it should go. It's not clearly defined what exception it's going to print the traceback for (the most recent one, but where was that?)